README.md

```
@@ -29,13 +31,13 @@ This software is distributed under the
 Please read [LICENSE](https://github.com/VilnaCRM-Org/frontend-ssr-template/blob/main/LICENSE) for information
 on the software availability and distribution.
 
-### Minimal installation
+### Minimal installation 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)